redbeams/src/main/resources/application.yml

```diff
@@ -204,6 +207,7 @@ cb:
     role.session.name: cdp-provisioning
     cf.template.new.path: templates/aws-cf-stack.ftl
     cf.template.newdb.path: templates/aws-cf-dbstack.ftl
+    rds.version: 10
```

Comment:
I think this is not used any more as the redbeams.db.postgres.major.version is used